cmd/openshift-install/create.go

```diff
@@ -323,6 +326,15 @@ func waitForInitializedCluster(ctx context.Context, config *rest.Config) error {
 		},
 	)
 
+	// If the CVO failed, print out the last failing operator statuses
+	if err != nil && clusterVersion != nil {
+		for _, c := range clusterVersion.Status.Conditions {
+			if c.Type == configv1.OperatorFailing {
+				logrus.Warn(c.Message)
```

Comment:
It only makes sense to print this if the `Failing` condition is `True`. Otherwise, it may make sense to print the message from the `Progressing` condition to cover the case where it is legitimately taking a long time for the cluster to initialize.

Reply:
Fixed the first part, punted on displaying progressing for now.